airtable/plugin.py

```diff
@@ -156,22 +165,29 @@ class AirtableRecord:
 
 
 def airtable_record_list(
-    base_id: AirtableBaseID,
```

Comment:
I don't know much about Airtable but it feels like base_id is probably more 'primary' than table_id ?